### DISCUSSION TITLE
SEG_LABEL not substituted with label. What is wrong?

### DISCUSSION THREAD

**Discussion by HamDeZign** [mifi/lossless-cut#2188] [General]
Love the application and am going through converted VHS recordings from the 80s. Give segments a good label as in "1983.06.21 Oslo (Fest) Øyvind John Silvia strolling down Karl Johan". My template for the filename is ${SEG_LABEL}${EXT}. Sometimes all segments are exported correctly, in other projects I get the issue where segment label is not used for filename (SEG_LABEL). Instead I get the label pluss CUT_FROM and CUT_TO info plus spill over labels from next segments. Then I can use SEG_SUFFIX, but then have to rename all exported files afterwards as they all start with a dash (-). What am I doing wrong and how to correct this?

**Comment by HamDeZign** [comment]
Oh, forgot: Am on Win 11 and running v3.62.0.0 and am cutting "mpeg - MPEG-1 System / MPEG program stream (detected" files...

**Reply by mifi** [reply]
Hi, it will revert to the default template if there is an error, like for example duplicate output names. if you open up the export confirmation dialog, you can see the error there

**Reply by HamDeZign** [reply]
There might be an error, but it's not duplicate filenames. It's just that I can't use the labels for filename, but suffix is ok, although that results in the same filename but with a dash (-) in front like: 
Fails with SEG_LABEL: 1983.06.21 Oslo (Fest) Øyvind John Silvia strolling down Karl Johan.mpg
OK with SEG_SUFFIX: -1983.06.21 Oslo (Fest) Øyvind John Silvia strolling down Karl Johan.mpg

This means, when I use "${SEG_LABEL}${EXT}" it failes to create the names, but if I use "${SEG_SUFFIX}${EXT}" it creates the filenames (with the annoying "-" in front of each filename (have to rename manually :- )

**Comment by HamDeZign** [comment]
Usually there are segments I don't want and in those cases I uncheck the segment. I just created 17 segments where about half of them are unchecked (meaning I don't want them exported). Anyways, I named the unchecked cuts with "1", "2", and so on. So that they would be unique if exported. The segments I wanted still got the segment names alla "1983.06.21 Oslo (Fest) Øyvind John Silvia strolling down Karl Johan". When I exported now, using the "${SEG_LABEL}${EXT}" template, it all worked fine - including the uncheck segments was not exported... Could that be the prob? I get duplicate names for the segments I don't want to export (unchecked)?

**Comment by mifi** [comment]
are you not getting any error under "Output file name template"? the duplicate check will only check the segments that are selected (checked), unchecked segments will be ignored.

I cannot reproduce your problem. Maybe if you provide a step-by-step process to reproduce it, i can help. see https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&projects=&template=2-bug.yml

or if you can share a .llc project file that reproduces the problem
